### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc0

### ISSUE THREAD

**Issue by AndreyRGW** [oobabooga/text-generation-webui#301]
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc0 in position 18: invalid start byte
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\textgen\lib\site-packages\gradio\routes.py", line 374, in run_predict
    output = await app.get_blocks().process_api(
  File "C:\ProgramData\Anaconda3\envs\textgen\lib\site-packages\gradio\blocks.py", line 1017, in process_api
    result = await self.call_function(
  File "C:\ProgramData\Anaconda3\envs\textgen\lib\site-packages\gradio\blocks.py", line 835, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "C:\ProgramData\Anaconda3\envs\textgen\lib\site-packages\anyio\to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "C:\ProgramData\Anaconda3\envs\textgen\lib\site-packages\anyio\_backends\_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "C:\ProgramData\Anaconda3\envs\textgen\lib\site-packages\anyio\_backends\_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "F:\WBC\text-generation-webui\modules\chat.py", line 337, in load_character
    data = json.loads(open(Path(f'characters/{_character}.json'), 'r', encoding='utf-8').read())
  File "C:\ProgramData\Anaconda3\envs\textgen\lib\codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xee in position 57: invalid continuation byte

**Comment by afgonczol** [comment]
What was the solution?

**Comment by AndreyRGW** [comment]
> What was the solution?

I just converted the text file with the character information to utf-8 using notepad++

**Comment by Verinoth** [comment]
I had a similar error. It happened once before but I either deleted installer_files or updated again and it went away. Next time I ran it, another UnicodeDecodeError similar to OP. Somewhere in the traceback it mentioned an "entry_points.txt" file so I searched through all of those, located one at:
"<my-install-dir>\text-generation-webui\installer_files\conda\pkgs\setuptools-68.0.0-py310haa95532_0\Lib\site-packages\setuptools-68.0.0-py3.10.egg-info\entry_points.txt"

Looked like a garbled mess in np++ but was ANSI encoding. Converted to UTF-8 and ooba ran fine at least one more time now.
